pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -181,6 +190,19 @@ BEGIN
         timescaledb.compress_orderby = 'time'
     ); $$, NEW.table_name);
 
+    SELECT SCHEMA_CATALOG.get_or_create_label_id('__name__', NEW.metric_name)
+    INTO STRICT label_id;
+
+    --note that because labels[0] is unique across partitions and UNIQUE(labels) inside partition, labels are guaranteed globally unique
```

Comment:
```suggestion
    --note that because labels[1] is unique across partitions and UNIQUE(labels) inside partition, labels are guaranteed globally unique
```